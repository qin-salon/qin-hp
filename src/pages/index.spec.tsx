import renderer from "react-test-renderer";

import Root from "./index.page";

describe("Root page", () => {
  it("matches snapshot", () => {
    const tree = renderer.create(<Root />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
