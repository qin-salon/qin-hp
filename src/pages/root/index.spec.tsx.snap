// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root page matches snapshot 1`] = `
<div
  className="c-fcgcOt"
>
  <div>
    <h2
      className="c-iMvRoz"
    >
      ABOUT
    </h2>
    <p
      className="c-cjhSLJ"
    >
      私達はテクノロジーを愛するコミュニティです。最高のプロダクトを創るために研鑽しています。 これが後に世界最大のテクノロジー集団になることは、私達以外まだ誰も知らない。
    </p>
  </div>
  <div>
    <h2
      className="c-iMvRoz"
    >
      PRODUCT
    </h2>
    <p
      className="c-cjhSLJ"
    >
      Coming Soon...
    </p>
  </div>
  <div>
    <h2
      className="c-iMvRoz"
    >
      COMPANY
    </h2>
    <div
      className="c-jNxTai"
    >
      <h3
        className="c-gKjkTc"
      >
        合同会社Qin
      </h3>
      <dl>
        <dt>
          事業
        </dt>
        <dd>
          インターネットビジネスの企画・開発・運営
        </dd>
      </dl>
      <dl>
        <dt>
          設立
        </dt>
        <dd>
          2021年7月
        </dd>
      </dl>
      <dl>
        <dt>
          代表
        </dt>
        <dd>
          島袋 光音
        </dd>
      </dl>
    </div>
  </div>
  <div>
    <h2
      className="c-iMvRoz"
    >
      CONTACT
    </h2>
    <p
      className="c-cjhSLJ"
    >
      <a
        className="c-hrMtea"
        href="https://twitter.com/shimabu_it"
        rel="noopener noreferrer"
        target="_blank"
      >
        代表のTwitter
      </a>
       
      宛にDMをお送りください
    </p>
  </div>
</div>
`;
